src/middleware.ts

```diff
@@ -1115,11 +1115,11 @@ const REDIRECTS: Redirect[] = [
   },
   {
     from: '/product/performance/event-detail/',
-    to: '/product/sentry-basics/concepts/tracing/event-detail/',
+    to: '/product/sentry-basics/concepts/key-terms/tracing/trace-view/',
   },
   {
     from: '/product/sentry-basics/tracing/event-detail/',
-    to: '/product/sentry-basics/concepts/tracing/event-detail/',
+    to: '/product/sentry-basics/concepts/key-terms/tracing/trace-view/',
```

Comment:
I think you also need to add a redirect from '/product/sentry-basics/concepts/tracing/event-detail/' to the trace view page.

Reply:
Great call!